src/service.ts.orig

```diff
@@ -0,0 +1,543 @@
+// Copyright (c) Jupyter Development Team.
```

Comment:
Should we add these `.orig` to the `.gitignore`?

Reply:
Yes we can do that. I'm going to remove the .orig added by mistake, I'll add the gitignore change in a dedicated commit.